types/net_matches.d.ts

```diff
@@ -0,0 +1,16 @@
+import { NetAllFields } from './net_fields';
+import { TriggerTypes } from './net_trigger';
+
+export type Matches = { [s in T]?: string };
+export type ToMatches<F> = Record<keyof F, string>;
+
+
+export type NetMatches = Matches & {
+};
```

Comment:
```suggestion
export type NetMatches = Matches;
```

Am I misunderstanding what this is doing? (Alternatively maybe there's no need for a separate `NetMatches` if it's the same as `Matches`?)

Reply:
This was leftover from when I was originally trying to have RegexTrigger have typing as well, before I hit the 50 union entries limit.